test-framework/junit5-internal/src/main/java/io/quarkus/test/QuarkusDevModeTest.java

```diff
@@ -70,7 +70,12 @@
 
     static {
         System.setProperty("java.util.logging.manager", "org.jboss.logmanager.LogManager");
```

Comment:
So this line is useless?

Reply:
Most likely yes, but I don't want to risk breaking anything - again :)

Reply:
Agreed.